### ISSUE TITLE
WCAG 1.3.1,WCAG 4.1.2: Ensures aria-hidden elements do not contain focusable elements (.offer-ul-list[data-slick-index="\31 "][aria-hidden="true"])

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#12]
#### Issue
`Ensures aria-hidden elements do not contain focusable elements` ([aria-hidden-focus](https://accessibilityinsights.io/info-examples/web/aria-hidden-focus))
#### Target application
[Personal Banking & Netbanking Services Online - ICICI Bank](https://www.icicibank.com/)
#### Element path
.offer-ul-list[data-slick-index="\31 "][aria-hidden="true"]
#### Snippet
`<div class="offer-ul-list slick-slide" data-slick-index="1" aria-hidden="true" tabindex="0" style="width: 280px;">`
#### How to fix
    Fix all of the following:
      Focusable content should have tabindex='-1' or be removed from the DOM
#### Environment
Chrome version 100.0.4896.127
This accessibility issue was found using Accessibility Insights for Web 2.31.1 (axe-core 4.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by manish10** [comment]
The title of this bug is different than the other bugs. All bug titles need to follow the same convention. I understand this one was created using accessibility insights. We can talk about what should be a good bug title convention.
